experimental/framework/get-container/package.json

```diff
@@ -29,7 +29,7 @@
     "@fluidframework/container-definitions": "^0.39.5",
     "@fluidframework/container-loader": "^0.44.0",
     "@fluidframework/core-interfaces": "^0.39.5",
-    "@fluidframework/driver-definitions": "^0.39.0",
+    "@fluidframework/driver-definitions": "^0.39.6",
```

Comment:
Might be worth separating PRs into one that bumps versions and everything else...